tests/Doctrine/Tests/DBAL/Functional/Driver/AbstractDriverTest.php

```diff
@@ -69,6 +69,6 @@ public function testReturnsDatabaseNameWithoutDatabaseNameParameter()
 
     protected static function getDatabaseNameForConnectionWithoutDatabaseNameParameter() : ?string
     {
-        return null;
+        return '';
```

Comment:
The test makes sense for the abstract driver because this is how a driver indicates that it's not connected to a DB. The test needs to be skilled in SQLite driver because it doesn't support databases. If not skipped, it'd fail with a non-in-memory database (the driver will return the database path, not null).